.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: '<what happen when you do on which document project>'
+labels: 'bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```

Comment:
Some issues I saw ever says only "sphinx-build" or "make html" command. But what we need is procedure from set up. It would be better to mention it here.

Reply:
updated: https://github.com/sphinx-doc/sphinx/pull/5942/files#diff-74c2649c999338ed7a2833d79d4b25c3R18